recipe/meta.yaml

```diff
@@ -11,10 +11,12 @@ source:
   sha256: bdcb529330730c4c364b79789620f33bafad8c2c854c1205ecc2d16511c0b690
 
 build:
-  number: 0
-  script: {{ PYTHON }} -m pip install . -vv
-  # OSX does not support fdatasync
-  skip: true  # [py<36 or osx]
+  number: 1
+  script:
+    # FIXME: Avoid 'ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]' error on osx builds
+    - export CFLAGS="$CFLAGS -Wno-implicit-function-declaration"  # [osx and x86]
```

Comment:
@mosquito as a heads up for the upstream project, this might become an issue for you when GitHub Actions updates to more modern versions of `clang`. [`macos-latest` current has `clang` `v15`](https://github.com/actions/runner-images/blob/8085da95cb9d8e1e2ce4e65a9dd0b1ccea1763bb/images/macos/macos-14-arm64-Readme.md#installed-software) where conda-forge is building with `clang` `v18` (and this is coming up).